common/v2/components/WalletUnlock/Web3ProviderInstall.tsx

```diff
@@ -6,7 +6,7 @@ import { DOWNLOAD_MYCRYPTO_LINK } from 'v2/config';
 
 import TrustWalletWEBP from 'common/assets/images/wallets/trust-3.webp';
 import CoinbaseWalletJPG from 'common/assets/images/wallets/coinbase.jpg';
-import MetamaskSVG from 'common/assets/images/wallets/metamask-2.svg';
+import MetamaskSVG from 'common/assets/images/wallets/metamask.svg';
```

Comment:
Are you sure about this?
The `.svg` files are really different

Reply:
I removed `metamask.svg` and renamed `metamask-2.svg` to `metamask.svg` - Since `metamask.svg` wasn't anywhere.

Reply:
Even better. Let me know when I can merge :)

Reply:
I haven't managed to look through them all yet, will mark it as ready when I've done so!